src/js/homepage_services.js

```diff
@@ -0,0 +1,21 @@
+
+function showTab(service) {
+
+  var i, tabcontent, tablinks;
+  tablinks = document.getElementsByClassName("section_services-tablink");
+  tabcontent = document.getElementsByClassName("section_services-tabcontent-wrapper");
+
+  for (i = 0; i < tabcontent.length; i++) {
+      tabcontent[i].style.display = "none";
+  }
+  
+  for (i = 0; i < tablinks.length; i++) {
+      tablinks[i].className = tablinks[i].className.replace(" open-tab", "");
+  }
+
+  document.getElementById(service).style.display = "flex";
+  // e.currentTarget.className += " open-tab";
```

Comment:
If you don't use remove it, and for this fragment of code in js exists `classList`